src/explore-education-statistics-admin/src/pages/release/pre-release/components/__tests__/PublicPreReleaseAccessForm.test.tsx

```diff
@@ -156,4 +159,22 @@ describe('PublicPreReleaseAccessForm', () => {
       });
     });
   });
+
+  test('does not render the Create / Edit button and shows a warning if the user does not have permission to update the release', () => {
+    render(
+      <TestConfigContextProvider>
+        <PublicPreReleaseAccessForm
+          canUpdateRelease={false}
+          preReleaseAccessList={testAccessList}
+          onSubmit={noop}
+        />
+      </TestConfigContextProvider>,
+    );
+    expect(
+      screen.getByText(
```

Comment:
Nice one!  This should probably be `queryByText` rather than `getByText` as `getByText` will blow up if it can't find the given text, whereas we really want `expect` to be doing the blowing up :)  `queryByText` is more forgiving and just returns null, allowing `expect(...).toBeInTheDocument()` to do its work.